opm/simulators/flow/KeywordValidation.cpp

```diff
@@ -129,14 +129,16 @@ namespace KeywordValidation
                     // Find the index number, which starts counting at one, so item_index + 1
                     const auto& item_properties = keyword_properties->second.find(item_index + 1);
                     if (item_properties != keyword_properties->second.end()) {
-                        // Validate the item, if it is partially supported.
-                        validateKeywordItem<T>(keyword,
-                                               item_properties->second,
-                                               keyword.size() > 1,
-                                               record_index,
-                                               item_index,
-                                               item.get<T>(0),
-                                               errors);
+                        if (item.hasValue(0) && !item.defaultApplied(0)) {
```

Comment:
I see no reason no to add the `!item.defaultApplied(0)` here - it is not totally insane with a situation where the default is actually not supported.

Reply:
Done.